src/webportal/src/app/job/job-view/fabric/job-detail/conn.js

```diff
@@ -90,8 +90,34 @@ export async function fetchJobRetries() {
   }
 }
 
-export async function fetchJobInfo() {
-  return wrapper(() => client.job.getJob(userName, jobName));
+export async function fetchJobInfo(attemptIndex) {
+  return wrapper(async () => {
+    const restServerUri = new URL(config.restServerUri, window.location.href);
+    const url = isNil(attemptIndex)
+      ? `${restServerUri}/api/v2/jobs/${userName}~${jobName}`
+      : `${restServerUri}/api/v2/jobs/${userName}~${jobName}/attempts/${attemptIndex}`;
```

Comment:
Align with RestServer, the WebUI also use below URL?
    const url = isNil(attemptIndex)
      ? `${restServerUri}/api/v2/jobs/${userName}~${jobName}`
      : `${restServerUri}/api/v2/jobs/${userName}~${jobName}/attempts/${attemptIndex}`;

Reply:
Could you explain more details?

Reply:
I mean url of the job detail page, after user selected one attempt index, may should change to like
${restServerUri}/api/v2/jobs/${userName}~${jobName}/attempts/${attemptIndex}

Reply:
So user can share the url of one specific attempt index, to others or save into onenote.

Reply:
Here the page use query parameter to do the same thing, you could share url like ```task-attempt.html?username=gusui&jobName=retry_test_3a23072a&jobAttemptIndex=2&taskRoleName=taskrole&taskIndex=0``` and get the same result.

The ```${restServerUri}/api/v2/jobs/${userName}~${jobName}/attempts/${attemptIndex}``` could not be done currently unless react router or nginx used.

Reply:
It should be a new specific page for the per attempt content

Reply:
So there will be 2 points in this discussion:
1. the web url follows ```${restServerUri}/api/v2/jobs/${userName}~${jobName}/attempts/${attemptIndex}``` pattern (hard)
2. add a new page for per attempt content (easy)

We could point out this requirement to scrum, however it might be not included in this pr.